### DISCUSSION TITLE
Velkommen til diskusjoner om spesifikasjoner og dokumentasjon av tillitsrammeverket!

### DISCUSSION THREAD

**Discussion by steinarnoem** [NorskHelsenett/Tillitsrammeverk#1] [Announcements]
## 👋 Velkommen!
  Vi bruker Diskusjoner som et sted hvor du kan knytte kontakt med andre som bidrar med spesifikasjoner og dokumentasjon. Vi håper at du:
  * Stiller spørsmål som du lurer på.
  * Deler ideer.
  * Deltar i diskusjoner.
  * Ønsker andre velkommen, og viser respekt. Dette er noe vi bygger sammen💪.

  For å komme i gang kan du gjerne legge inn en kommentar under, hvor du introduserer deg selv og forteller litt om hvem du er og hvorfor du ønsker å bidra i dette arbeidet.

**Comment by steinarnoem** [comment]
Hei! Jeg heter Steinar Noem. Jeg er innleid hos NHN hvor jeg jobber med å etablere tillitsrammeverk for deling av helseopplysninger innad i norsk helsesektor.

**Comment by richardhus** [comment]
Hei!
Jeg heter Richard Husevåg og er innleid hos Helse Sør-Øst/Sykehuspartner som hovedansvarlig arkitekt til prosjektet for kjernejournal dokumentdeling.

Jeg motiveres av å jobbe med IKT-løsninger som gir verdi for helsepersonell og pasienter.

Arbeidet med et tillitsrammeverk anser jeg å være et grunnleggende fundament for at man skal kunne åpne for enklest og bredest mulig deling av helseopplysninger mellom virksomheter, avgrenset til formålet å kunne gi pasienter god og forsvarlig helsehjelp. Eventuelt andre legitime formål som forskning og undervisning. Samtidig er jeg opptatt av at vi må ivareta tilliten vi som innbyggere har til norsk helsevesen i prosessen, herunder å ivareta forventninger om fortrolighet som kan ha blitt gitt før delingsløsninger basert på maskinell/automatisert tilgangskontroll til et bredt utvalg av forhåndsvurderte helseopplysninger var en mulighet.

**Comment by OK-SverreM** [comment]
Hei!
Jeg heter Sverre M. Jensen og er engasjert som virksomhetsarkitekt hos Helseetaten i Oslo kommune (innleid fra Bouvet). Jeg deltar i satsingen kalt Helsedata i Oslo, som er et samarbeid mellom Helseetaten og Oslo Origo, sammen med bl.a. Eirik Broen og Liv Rossow. 

Jeg er opptatt av å legge til rette for og realisere deling av informasjon mellom ulike aktører i helsesektoren. Ved siden av arbeidet med Tillitsmodell & dokumentdeling, er jeg også involvert i Oslo kommunes engasjement inn mot andre nasjonale delingstjenester, slik som Pasientens prøvesvar og Datadeling for Digital hjemmeoppfølging.

**Comment by jndid** [comment]
Jon Didriksen, innleid i NHN (fra Computas AS) som funksjonell arkitekt for Velferdsteknologisk knutepunkt (VKP). Vi skal jobbe med deling av data innen digital hjemmeoppfølging mellom aktører og behandlingsnivåer.  Da håper vi å kunne benytte oss mest mulig av det som allerede er definert i tillitsrammeverket. Jeg vil også gjerne forstå litt av vurderingene som ligger bak - hvor mye/lite informasjon skal utveksles og hvorfor?

**Comment by annekristinstrand** [comment]
Hei!
Anne Kristin Strand, arkitekt i Sykehuspartner. Deltar i prosjekt kjernejournal dokumentdeling i HSØ.

**Comment by stvgr** [comment]
Hei!
Jeg heter Eva Tone Fosse og jobber i Stavanger kommune med arkitektur og utvikling. 
Jeg er kommunens kontaktpunkt for dokumentdeling, og kommer til å delta her på vegne av kommunen.
